chia/consensus/block_creation.py

```diff
@@ -227,6 +230,10 @@ def create_foliage(
             generator_hash = std_hash(new_block_gen.program)
 
         generator_refs_hash = bytes32([1] * 32)
```

Comment:
Is this `generator_refs_hash` value documented somewhere compared to the value we compute when we have a `generator_block_heights_list` below?

Reply:
I don't know, I wouldn't be surprised if it's documented in the green paper. The most authoritative documentation is in `block_body_validation.py` though.